performance_regression_test.py

```diff
@@ -672,3 +677,136 @@ def test_timeseries_bench(self):
         self.display_results(results, test_name='test_timeseries_read_bench')
         self.check_regression()
         self.kill_stress_thread()
+
+    def test_mixed_with_stairs(self):  # pylint: disable=too-many-locals
```

Comment:
i thought you said you will rename it...
it could be called `test_mixed_gradual_increase_load` (or anything else you think fits better)